docs/source/zero-code-change.rst

```diff
@@ -163,10 +166,31 @@ data before measuring runtime on a full-scale dataset.
 
 6. Will I run out of GPU memory if I use ``cuml.accel``?
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-``cuml.accel`` will use CUDA `managed memory <https://developer.nvidia.com/blog/unified-memory-cuda-beginners/>`_ for allocations on NVIDIA GPUs. This means that host memory can be used to augment GPU memory, and data will be migrated automatically as necessary. This does not mean that ``cuml.accel`` is entirely impervious to OOM errors, however. Very large datasets can exhaust the entirety of both host and device memory. Additionally, if device memory is heavily oversubscribed, it can lead to slow execution. ``cuml.accel`` is designed to minimize both possibilities, but if you observe OOM errors or slow execution on data that should fit in combined host plus device memory for your system, please `report it <https://github.com/rapidsai/cuml/issues/new?template=bug_report.md>`_, and the RAPIDS team will investigate.
+
+When possible, ``cuml.accel`` will enable `managed memory
+<https://developer.nvidia.com/blog/unified-memory-cuda-beginners/>`_ for
+allocations on NVIDIA GPUs. This means that host memory can be used to augment
+GPU memory, and data will be migrated automatically as necessary. This does not
+mean that ``cuml.accel`` is entirely impervious to OOM errors, however. Very
+large datasets can exhaust the entirety of both host and device memory.
+Additionally, if device memory is heavily oversubscribed, it can lead to slow
+execution. ``cuml.accel`` is designed to minimize both possibilities, but if
+you observe OOM errors or slow execution on data that should fit in combined
+host plus device memory for your system, please `report it
+<https://github.com/rapidsai/cuml/issues/new?template=bug_report.md>`_, and the
+RAPIDS team will investigate.
```

Comment:
This paragraph is mostly the same as before, just line wrapped. The note below is the main addition.

Reply:
We should probably just install an auto-formatter that automates desired line wrapping.